webpack.config.js

```diff
@@ -79,19 +80,19 @@ const terraApplicationConfig = (env = {}) => ({
       titleConfig: {
         title: 'Terra Dev Site - Extended',
       },
-      additionalSearchDirectories: [
-        path.resolve(process.cwd(), 'node_modules', 'terra-list', 'lib', 'terra-dev-site'),
-      ],
+      // additionalSearchDirectories: [
```

Comment:
remove?

Reply:
It's kind of a reminder. I need to backfill exercising this part of the API with something else. This is a victim of the new strategy of breaking dev sites out from individual packages and into docs packages per mono repo. Eventually terra-list won't have anything to find.

Reply:
not a big deal, but a todo comment in the interim would be helpful

Reply:
I went ahead and took care of the todo :)